src/Makefile

```diff
@@ -0,0 +1,14 @@
+NAME = SineGenerator
+
+TYPE = APP
+
+APP_MIME_SIG = application/x-vnd-WakeUp
```

Comment:
Is this Makefile really for WakeUp or for SineGenerator?

Reply:
How does it look like now?